### DISCUSSION TITLE
ResizeObserver not working in lit element

### DISCUSSION THREAD

**Discussion by jitendraP-ashutec** [lit/lit#3151] [Q&A] [answered]
I am trying to use ResizeObserver  but its not working.

`
 const resizeObserver = new ResizeObserver(entries => {
        console.log(entries);
      });

  resizeObserver.observe(this);
`

I want to perform calculation based on current lit element.

**Comment by sorvell** [comment, marked as answer]
It'd be great to have a repro, but the likely culprit is not setting the element's display to something other than the custom element default of `inline`. Setting this to `block` should make it work, e.g. `:host { display: block; }`.

Here's a working example, click the red box and look at the console for output: https://lit.dev/playground/#gist=8e9c87a92273cd6194cc1abf5cf74159.

**Reply by jitendraP-ashutec** [reply]
Thanks @sorvell
